### PR TITLE
hybrid-overlay: enable host to access hybrid overlay nodes

### PR DESCRIPTION
**Pull request by dcbw**
Set up a route via the hybrid overlay port IP address through
the management port device to hybrid overlay nodes. Also add
a reply flow to send the traffic back to the management port.
As a bonus, makes services work.

@JacobTanenbaum

### CODE REVIEW

**Comment by dcbw** [comment]
Coveralls coverage decrease is likely due to the fact that we must sudo exec the hybrid overlay tests now, and coverage detection can't look into that.

**Comment by JacobTanenbaum** [comment]
\lgtm

File: go-controller/hack/test-go.sh [hunk 1]
```diff
@@ -23,10 +23,12 @@ function testrun {
     go test -mod vendor ${args}
 }
 
+root_pkgs=("github.com/ovn-org/ovn-kubernetes/go-controller/pkg/node" "github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/controller")
+
 i=0
 for pkg in ${PKGS}; do
     # This package requires root
```
Comment (danwinship) [inline go-controller/hack/test-go.sh:30]: reword the comment and move it to above the variable declaration?

(also, boo! unit tests shouldn't require being root. grump grump grump.)

Comment (dcbw) [reply go-controller/hack/test-go.sh]: Fixed.

**Pull request merged**